infrastructure/experience/cert.tf

```diff
@@ -0,0 +1,19 @@
+module "wellcomecollection_cert" {
```

Comment:
Previously this stack had a hard-coded value for the cert - this imports it into terraform state.